docs/articles/documentation/continuous-integration/bitbucket-pipelines.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: docs
+title: Run Tests in Bitbucket Pipelines
+permalink: /documentation/continuous-integration/bitbucket-pipelines.html
+---
+# Run Tests in Bitbucket Pipelines CI
+
+You can automatically run TestCafe tests as a part of your build process on [Bitbucket Pipelines CI](https://bitbucket.org/product/features/pipelines).
```

Comment:
You can run TestCafe tests as a part of your build process on [Bitbucket Pipelines CI](https://bitbucket.org/product/features/pipelines).